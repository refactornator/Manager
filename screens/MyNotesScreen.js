import React from 'react';
import styled from 'styled-components/native';

import NotesScreen from './NotesScreen';

export default props => {
  return (
    <Container>
      <NotesScreen {...props} report={null} keyboardVerticalOffset={0} />
    </Container>
  );
};

const Container = styled.View`
  flex: 1;
  padding-top: 30px;
  background-color: #f4f4f4;
`;
