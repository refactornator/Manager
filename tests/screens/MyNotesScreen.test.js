import React from 'react';

import { shallow } from 'enzyme';

import MyNotesScreen from '../../screens/MyNotesScreen';

it("renders the manager's notes screen", () => {
  const render = shallow(<MyNotesScreen />);
  expect(render).toMatchSnapshot();
});
