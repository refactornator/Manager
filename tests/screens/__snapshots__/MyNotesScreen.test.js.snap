// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the manager's notes screen 1`] = `
<Styled(View)>
  <NotesScreen
    keyboardVerticalOffset={0}
    report={null}
  />
</Styled(View)>
`;
